README.md

```diff
@@ -247,6 +247,15 @@ require'sniprun'.setup({
     -- "LongTempFloatingWindow",  -- "same as above, but only long results. To use with VirtualText__
     -- "Terminal"                 -- "display results in a vertical split
     },
+    
+  -- customize highlight groups
+  SniprunColors = {
```

Comment:
This should appear in the default lua configuration, and also, all configuration keys are lowercase, for consistency i'd like it to stay that way.

This config works a bit differently than most lua plugins:
sniprun overwrites the default configuration on a key by key basis: this means you don't have to copy the full configuration in order to modify just one key.